### DISCUSSION TITLE
Copilot is encountering temporary issues(click for details)

### DISCUSSION THREAD

**Discussion by FengZi-lv** [community/community#43793] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I just finished registering copilot in vscode, it prompts `Copilot is encountering temporary issues(click for details)`

Then I looked at the "Output" about copilot in vscode
```
[ERROR] [fetch] [2023-01-10T01:22:12.648Z] Unhandled status from server:,404,{"error":{"message":"engine \"copilot-logit\" not available"}}
```

**Comment by davecheney** [comment]
I’m sorry you are experiencing problems. Can you please 

1. check your copilot settings; extension -> copilot -> edit extension.json; remove any reference to “copilot-login”
2. Upgrade your client to the latest available. 

Thank you

**Reply by henryph24** [reply]
problem persists? any other recommendations? It happened since the latest subscription renewal

**Reply by henryph24** [reply]
I did not know of the new policy regarding the monthly reverification of student status. So when it is suggested that I can have 60 days trial, I immediately registered. Afterwards, the free package get renewed. Just wondering if this can be part of the problem from my end

**Reply by henrzven** [reply]
I don't have any reference to copilot-login, error still persists

**Reply by AndresDiagoM** [reply]
yeah, It happened since the latest subscription renewal, using student package

**Reply by Marubi21** [reply]
Since the last renewal it works as it wants. With simple commands it sometimes works but in most cases the above error is shown. Anyone know how to achieve this?

**Reply by AndresDiagoM** [reply]
i`m also using the code GPT extension, is there any problem while working with copilot too?

**Comment by opuu** [comment]
Same problem here, everything is up to date.
Entering a new line sometimes fix it then again the same problem, restarting vscode is not working as well.

Edit:

This is not a bug, it seems to be intentional :)

```log
[ERROR] [fetch] [2023-01-10T17:42:45.818Z] Unhandled status from server:,429,{"error":{"message":"rate limit exceeded","internal_message":"rate limit exceeded for plan FREE_EDUCATIONAL","type":"client_error"}}
```

**Comment by Stepashka20** [comment]
Same problem all day:(

**Comment by Usama008** [comment]
Same problem as well, it works for like one or two lines at most then it stops.

**Comment by FrancyMak** [comment]
Same problem here

**Reply by nxm** [reply]
same

**Reply by Bhartendu-Kumar** [reply]
same

**Comment by catalin-damian** [comment]
Copilot is built on the OpenAI Codex/infrastructure. Unfortunately, OpenAI has been under an enormous load since they released ChatGPT. I firmly believe this is the reason why GitHub Copilot is fussy today.

**Comment by ryanhaticus** [comment]
Not the only one!

**Comment by tomkortjak** [comment]
Also encountering this problem

**Comment by ReaVNaiL** [comment]
Answered by Support Team in #43673

**Comment by abdellatif-laghjaj** [comment]
Saame issue here, it compile for some lines of code, and then it stopped !

**Comment by cuckito** [comment]
Same here

**Comment by Bhartendu-Kumar** [comment]
same issue, its frustratingly frequent to show this error

**Comment by Alirez14** [comment]
why can Microsoft not do a thing right? we pay for copilot and they rate limits us

**Comment by dazmy** [comment]
I thought it was just me

**Comment by Heath123** [comment]
I'm getting

```
[ERROR] [fetch] [2023-01-12T12:20:25.142Z] Unhandled status from server:,429,{"error":{"message":"rate limit exceeded","internal_message":"rate limit exceeded for plan FREE_EDUCATIONAL","type":"client_error"}}
```

Copilot seems to be showing this error most of the time, isn't that rate limit a bit extreme?

**Reply by Akash1134** [reply]
Hi @FengZi-lv ,

This week, we've had to take down some of our Copilot resources for maintenance. While we did our best to source extra capacity elsewhere, we've been forced to limit some free users for the period of time where one of our busiest models is out of commission.

We expect that this will cease to impact your use of Copilot after the 16th of January. In the meantime, you will have less frequent suggestions, but you should still be getting suggestions - just with rate limits in between. You may see these errors specifically if you look in the output pane or logs for your IDE.

We are also aware that some students are still experiencing issues with redeeming and accessing Copilot and that the team are currently working to resolve this. We'll update the community when we have more information on that incident in a separate discussion.

Thanks for your patience and apologies for the inconvenience caused.

**Comment by opuu** [comment]
This problem seems to be fixed now.

**Comment by wilsonborba** [comment]
I'm facing the same issue: 

I've already uninstalled and reinstalled, 
verified that my bill is paid, 
closed and reopened VSCode,
 tried different languages, 
 checked the settings to ensure autocomplete is enabled for all files and languages,
  and confirmed that autocomplete is enabled.
  
  While the Copilot chat works normally, autocomplete does not. I'm not sure what I'm missing.

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/108008
